android/app/src/main/res/layout/activity_manual_control.xml

```diff
@@ -42,26 +42,34 @@
         app:layout_constraintTop_toTopOf="parent"></ImageView>
 
     <TextView
-        android:id="@+id/textView2"
+        android:id="@+id/speedHeader"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginStart="24dp"
-        android:layout_marginLeft="24dp"
-        android:layout_marginTop="4dp"
-        android:text="Speed"
-        app:layout_constraintStart_toStartOf="@+id/joystickView2"
-        app:layout_constraintTop_toTopOf="@+id/joystickView2" />
+        android:layout_marginStart="4dp"
+        android:layout_marginLeft="4dp"
+        android:layout_marginBottom="8dp"
+        android:fontFamily="serif"
+        android:text="Speed:"
+        android:textSize="20sp"
+        android:textColor="@color/teal_700"
+        android:textStyle="bold"
+        app:layout_constraintBottom_toTopOf="@+id/joystickView2"
+        app:layout_constraintStart_toStartOf="@+id/imageView" />
```

Comment:
I think it would be good to add another constraint  to the bottom of the camera view image, so that the angle header and speed header don't overlap with the camera view on smaller devices.

Reply:
That a good point! That would also make sure it would follow the cameraview even though the position of the camera changes. Gonna change this 👍